src/ring_theory/witt_vector/frobenius.lean

```diff
@@ -57,7 +57,7 @@ open_locale big_operators
 variables (p)
 include hp
 
-/-- The rational polynomials that give the coefficients of `frobenius x`,
+/-- The rational funcions that give the coefficients of `frobenius x`,
```

Comment:
In this case, it's actually talking about `polynomial ℚ`, so this was a false positive.
```suggestion
/-- The rational polynomials that give the coefficients of `frobenius x`,
```